src/SourceBuild/tarball/content/eng/Versions.props

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- SDK/Runtime Version Information -->
+  <PropertyGroup>
+    <MajorVersion>5</MajorVersion>
```

Comment:
I think we can remove these - I think they should get propagated from installer.  Otherwise probably want to update.